src/tools.cpp

```diff
@@ -1162,3 +1165,30 @@ const char* getReturnMessage(ReturnValue value)
 			return "Sorry, not possible.";
 	}
 }
+
+bool loadDirectory(const std::string directory)
```

Comment:
ok that certainly makes sense, will rename it to "loadDirectoryLuaFiles" or something like that.